### ISSUE TITLE
How do i get the logits from the ASR model

### ISSUE THREAD

**Issue by sciai-ai** [flashlight/flashlight#1044]
I am using the flashlight/build/bin/asr/fl_asr_test command and able to get the output. Is is possible to get the logits that have been predicted by the model?

**Comment by jacobkahn** [comment]
@sciai-ai — we use `emission` interchangeably with logits. Emissions are available by changing the source here: https://github.com/flashlight/flashlight/blob/main/flashlight/app/asr/Test.cpp#L308 — `rawEmissions` is a GPU tensor with emissions, and `emissions` is the `vector<float>` on the CPU.
